SmartDeviceLink/SmartDeviceLink.h

```diff
@@ -347,11 +347,15 @@ FOUNDATION_EXPORT const unsigned char SmartDeviceLinkVersionString[];
 #import "SDLPermissionConstants.h"
 #import "SDLPermissionManager.h"
 
-// Show
+// Screen
 #import "SDLScreenManager.h"
 #import "SDLSoftButtonObject.h"
 #import "SDLSoftButtonState.h"
 
+#import "SDLMenuManager.h"
+#import "SDLMenuCell.h"
+#import "SDLVoiceCommand.h"
+
```

Comment:
SDLMenuManager.h , SDLMenuCell.h,  and SDLVoiceCommand.h, all need to be added to the **.podspec** files.